webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/esbuild.js

```diff
@@ -0,0 +1,45 @@
+// Load the go wasm.
+require("./go-wasm");
+
+class EsBuild extends Go {
+  constructor() {
+    super();
+    this.wasm = undefined;
+    this.workDir = "/app";
+    this.user = "esbuild";
+    this.homeDir = "/home/esbuild";
+    this.env.USER = this.user;
+    this.env.HOME = this.homeDir;
+    // process.cwd = () => this.workDir;
+  }
+
+  async load() {
+    const response = await fetch("esbuild.wasm");
```

Comment:
how does this work? "esbuild.wasm" is externalized here?

Reply:
esbuild.wasm is external. The prebuild plugin moves it into dist and public.